packages/zowe-explorer/src/Profiles.ts

```diff
@@ -885,7 +885,7 @@ export class Profiles extends ProfilesCache {
         }
         const usingSecureCreds = SettingsConfig.getDirectValue(globals.SETTINGS_SECURE_CREDENTIALS_ENABLED);
         const profInfo = await this.getProfileInfo();
-        if (profInfo.usingTeamConfig && usingSecureCreds) {
+        if (!imperative.ProfileInfo.onlyV1ProfilesExist && usingSecureCreds) {
```

Comment:
Minor optimization: Checking `usingSecureCreds` first will short-circuit the call to `ProfileInfo.onlyV1ProfilesExist` when secure credentials are not enabled.